pxr/usd/usdLux/lightAPI.h

```diff
@@ -359,10 +421,119 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
     // --------------------------------------------------------------------- //
     // NORMALIZE 
     // --------------------------------------------------------------------- //
-    /// Normalizes power by the surface area of the light.
+    /// Normalizes the emission such that the power of the light
+    /// remains constant while altering the size of the light, by dividing the
+    /// luminance by the world-space surface area of the light.
+    /// 
     /// This makes it easier to independently adjust the power and shape
```

Comment:
should be `adjust the brightness and size` per @anderslanglands

Reply:
Done: https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/2533cb0a9bbfed73a5e2c970c67d8b2f91a27b52#diff-bdeeccf2a8491e83067bc3c1e20e5a696612da9b31d1e3b7e6e7fc33b3592ec7R256